backends/aerospike.go

```diff
@@ -107,31 +111,36 @@ func (a *AerospikeBackend) Get(ctx context.Context, key string) (string, error)
 func (a *AerospikeBackend) Put(ctx context.Context, key string, value string, ttlSeconds int) error {
 	asKey, err := a.client.NewUuidKey(a.cfg.Namespace, key)
 	if err != nil {
-		return formatAerospikeError(err)
+		return classifyAerospikeError(err)
 	}
 
 	if ttlSeconds == 0 {
 		ttlSeconds = a.cfg.DefaultTTL
 	}
 
 	bins := as.BinMap{binValue: value}
-	policy := &as.WritePolicy{Expiration: uint32(ttlSeconds)}
+	policy := &as.WritePolicy{
+		Expiration:         uint32(ttlSeconds),
+		RecordExistsAction: as.CREATE_ONLY,
```

Comment:
This is also defined in `NewAerospikeBackend`, are both needed?

Reply:
Good catch, will remove from `NewAerospikeBackend`, as `ttlSeconds` must get included in the write policy here.